app/src/main/java/com/cryart/sabbathschool/ui/about/AboutActivity.kt

```diff
@@ -48,11 +51,35 @@ class AboutActivity : SSColorSchemeActivity() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        return if (item.itemId == android.R.id.home) {
-            finishAfterTransition()
-            true
-        } else {
-            super.onOptionsItemSelected(item)
+        return when (item.itemId) {
+            android.R.id.home -> {
+                finishAfterTransition()
+                true
+            }
+            R.id.ss_action_share -> {
+                val shareIntent = ShareCompat.IntentBuilder(this)
+                    .setType("text/plain")
+                    .setText(
+                        String.format(
```

Comment:
Use Kotlin string template instead of format

Reply:
Done